index.html

```diff
@@ -0,0 +1,141 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <section class="my-project">
+      <h1>Projects</h1>
+      <img src="images/Rectangle 21.png" alt=""  class="project1"/>
+      <div class="works">
+        <h2>Project name goes here</h2>
+        <div class="languages">
+          <button type="button" class="mark-up">HTML/CSS</button>
+          <button type="button" class="react">Ruby on Rails</button>
+          <button type="button"class="responsive">JavaScript</button>
+        </div>
+          <button type="button" class="show-works">
+          See this work here &#x2192
+          </button>
+      </div>
+
+      <div class="project-2">
+        <img src="images/Rectangle 22.png" alt="" class="project2">
+        <div class="work-2">
+<h2 class="second-project">Project name goes here</h2>
+ <div class="languages-2">
+          <button type="button" class="mark-up-2">HTML/CSS</button>
+          <button type="button" class="react-2">Ruby on Rails</button>
+          <button type="button" class="responsive-2">JavaScript</button>
+        </div>
+        </div>
+          <button type="button" class="show-works-2">
+          See this work here &#x2192
+          </button>
+        </div>
+      </div>
+
+    <div class="project-3">
+      <img src="images/project 3.png" alt="" class="project3">
+      <div class="work-3">
+ <h2 class="third-project">Project name goes here</h2>
+<div class="languages-3">
+          <button type="button" class="mark-up-3">HTML/CSS</button>
+          <button type="button" class="react-3">Ruby on Rails</button>
+          <button type="button" class="responsive-3">JavaScript</button>
+        </div>
```

Comment:
As requested on the project requirements:
![image](https://user-images.githubusercontent.com/42799579/198789937-d01e5945-cc75-4e6b-aaca-9f52caae49e3.png)

Please take into account that the technologies should be inside an `ul` and listed as `li` instead of buttons. You could have something like this:
```html
<ul class="technologies">
    <li>HTML/CSS</li>
    <li>JavaScript</li>
    <li>...</li>
</ul>
```